ui/main.60

```diff
@@ -151,6 +151,9 @@ CargoUI := Window {
 
     property <string> status: "Cargo UI";
     property <[string]> packages;
+    property <int> packages_count;
```

Comment:
Why do we need this roperty.  Instead, we could set `allow_package_selection` directly from the rust code.

Reply:
Sure, I'll do that.